projects/1.18/assets/ydms-wizarding-world-of-harry-potter/harrypotter/lang/zh_cn.json

```
@@ -0,0 +1,117 @@
+{
+"effect.harrypotter.wingardium_leviosa_effect": "羽加迪姆·勒维奥萨",
+"screen.harrypotter.hotbar": "快捷栏",
+"screen.harrypotter.spellbook": "法术书",

```

```suggestion
"screen.harrypotter.spellbook": "魔法书",
```
```suggestion
"screen.harrypotter.spellbook": "咒语书",
```
一般来说，哈利波特里不使用`法术`一词，而是用`符咒`或`魔咒`对应`spell`  
不过也没有太严格的对应关系，`魔法书`也是可以接受的